pkg/mcs/tso/server/server.go

```diff
@@ -39,7 +39,7 @@ type Server struct {
 
 // TODO: Implement the following methods defined in bs.Server
 
-// Name returns the unique etcd Name for this server in etcd cluster.
+// Name returns the unique etcd Name for this server in etcd cluster
```

Comment:
Unnecessary change.

Reply:
I don't remember I changed it. Will change it back.

Reply:
changed back.